### PR TITLE
Python tracing example

### PR DESCRIPTION
**Pull request by wiliamsouza**
Update tracing example to work with version `0.4a0` and `master` branch

### CODE REVIEW

**Comment by wiliamsouza** [comment]
I signed it

File: content/docs/python/tracing.md [hunk 1]
```diff
@@ -20,30 +20,31 @@ From there, you should be able to use opentelemetry as per the following:
...
 from opentelemetry.context import Context
-from opentelemetry.sdk.trace import Tracer
+from opentelemetry.sdk.trace import TracerSource
 from opentelemetry.sdk.trace.export import ConsoleSpanExporter
 from opentelemetry.sdk.trace.export import SimpleExportSpanProcessor
 
-trace.set_preferred_tracer_implementation(lambda T: Tracer())
-tracer = trace.tracer()
-tracer.add_span_processor(
+trace.set_preferred_tracer_source_implementation(lambda T: TracerSource())
```
Comment (MikeGoldsmith) [inline content/docs/python/tracing.md]:
```suggestion
trace.set_preferred_tracer_provider_implementation(lambda T: TracerProvider())
```

File: content/docs/python/tracing.md [hunk 2]
```diff
@@ -20,30 +20,31 @@ From there, you should be able to use opentelemetry as per the following:
...
+tracer = trace.get_tracer(__name__)
+trace.tracer_source().add_span_processor(
```
Comment (MikeGoldsmith) [inline content/docs/python/tracing.md]:
```suggestion
trace.tracer_provider().add_span_processor(
```

File: content/docs/python/tracing.md [hunk 3]
```diff
@@ -20,30 +20,31 @@ From there, you should be able to use opentelemetry as per the following:
 ```python
 from opentelemetry import trace
 from opentelemetry.context import Context
-from opentelemetry.sdk.trace import Tracer
+from opentelemetry.sdk.trace import TracerSource
```
Comment (MikeGoldsmith) [inline content/docs/python/tracing.md]: TracerSource has been renamed to TracerProvider - [opentelemetry-python/441](https://github.com/open-telemetry/opentelemetry-python/pull/441)

```suggestion
from opentelemetry.sdk.trace import TracerProvider
```

**Comment by wiliamsouza** [comment]
> Looks good - just few small suggestions.

With your suggestions we have:
```
from opentelemetry import trace
from opentelemetry.context import Context
from opentelemetry.sdk.trace import TracerProvider
from opentelemetry.sdk.trace.export import ConsoleSpanExporter
from opentelemetry.sdk.trace.export import SimpleExportSpanProcessor

trace.set_preferred_tracer_provider_implementation(lambda T: TracerProvider())
tracer = trace.get_tracer(__name__)
trace.tracer_provider().add_span_processor(
    SimpleExportSpanProcessor(ConsoleSpanExporter())
)

with tracer.start_as_current_span("foo"):
    with tracer.start_as_current_span("bar"):
        with tracer.start_as_current_span("baz"):
            print(Context)

``` 

It is not working for  version `0.4.0`:

```
$ git branch
  master
* v0.4.0

$ python t.py
Traceback (most recent call last):
  File "t.py", line 3, in <module>
    from opentelemetry.sdk.trace import TracerProvider
ImportError: cannot import name 'TracerProvider' from 'opentelemetry.sdk.trace' (/home/wiliam/Development/opentelemetry-python/opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py)
```

It works with master branch:

```
$ git branch 
* master
  v0.4.0

$ python t.py 
<class 'opentelemetry.context.context.Context'>
Span(name="baz", context=SpanContext(trace_id=0xea9608605d7057be51e01a54fd12f8b8, span_id=0xe993426c87480466, trace_state={}), kind=SpanKind.INTERNAL, parent=Span(name="bar", context=SpanContext(trace_id=0xea9608605d7057be51e01a54fd12f8b8, span_id=0x16fe0a23cadf11cc, trace_state={})), start_time=2020-03-04T16:43:45.665060Z, end_time=2020-03-04T16:43:45.665087Z)
Span(name="bar", context=SpanContext(trace_id=0xea9608605d7057be51e01a54fd12f8b8, span_id=0x16fe0a23cadf11cc, trace_state={}), kind=SpanKind.INTERNAL, parent=Span(name="foo", context=SpanContext(trace_id=0xea9608605d7057be51e01a54fd12f8b8, span_id=0xf36f63ffe9cf4bee, trace_state={})), start_time=2020-03-04T16:43:45.665044Z, end_time=2020-03-04T16:43:45.665205Z)
Span(name="foo", context=SpanContext(trace_id=0xea9608605d7057be51e01a54fd12f8b8, span_id=0xf36f63ffe9cf4bee, trace_state={}), kind=SpanKind.INTERNAL, parent=None, start_time=2020-03-04T16:43:45.665013Z, end_time=2020-03-04T16:43:45.665274Z)
```

But the docs uses the latest release so the examples should work with that to avoid confusion.

**Comment by mtwo** [comment]
Added @c24t as a reviewer

**Comment by mauriciovasquezbernal** [comment]
@wiliamsouza FYI, I think the example should be aligned with the last released version, that's it: https://github.com/open-telemetry/opentelemetry-python/tree/v0.4.1#tracing

File: content/docs/python/tracing.md [hunk 4]
```diff
@@ -20,30 +20,31 @@ From there, you should be able to use opentelemetry as per the following:
...
+trace.tracer_source().add_span_processor(
     SimpleExportSpanProcessor(ConsoleSpanExporter())
 )
-with tracer.start_as_current_span('foo'):
-    with tracer.start_as_current_span('bar'):
-        with tracer.start_as_current_span('baz'):
+with tracer.start_as_current_span("foo"):
+    with tracer.start_as_current_span("bar"):
+        with tracer.start_as_current_span("baz"):
             print(Context)
```
Comment (mauriciovasquezbernal) [inline content/docs/python/tracing.md]: Would you mind updating this like as well?

Comment (wiliamsouza) [reply content/docs/python/tracing.md]: Done. Using `print("Hello world from OpenTelemetry Python!")`

**Pull request merged**